python/ray/tune/progress_reporter.py

```diff
@@ -169,11 +172,13 @@ class TuneReporterBase(ProgressReporter):
 
     def __init__(
         self,
+        *,
```

Comment:
why this?

Reply:
This class has 11 arguments in no intuitive order. It's unlikely users will call it with args and not kwargs, and it's error prone when dealing with derived classes. Instead we should force users to use kwargs. This will also enable us to refactor this in the future.

Reply:
Ah got it. so this is to force zero positional argument :)